apps/public-docsite-v9/src/Concepts/Migration/FromV8/ThemeColors/colorMap.ts

```diff
@@ -135,6 +135,8 @@ export const alias = {
   colorCompoundBrandForeground1Pressed: 'brand[60]',
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 9 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27791&commitId=7e2af887ed85dd2346db8bbdcafa33256263e3e3&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1924 screenshots unchanged, 0 screenshots with different dimensions and 9 screenshots with visible difference.


  <details><summary><b>unknown</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Badge Converged.tint - Dark Mode.chromium.png|217| Changed|
|Badge Converged.tint - High Contrast.chromium.png|393| Changed|
|DatePicker Compat.marked dates - High Contrast.default.chromium.png|12| Changed|
|Table layout flex.multiselect (checked) - brand - Dark Mode.chromium.png|24857| Changed|
|Table layout flex.multiselect (mixed) - brand - Dark Mode.chromium.png|12405| Changed|
|Table layout flex.single select (checked) - brand - Dark Mode.chromium.png|6200| Changed|
|Table layout table.multiselect (checked) - brand - Dark Mode.chromium.png|24857| Changed|
|Table layout table.multiselect (mixed) - brand - Dark Mode.chromium.png|12405| Changed|
|Table layout table.single select (checked) - brand - Dark Mode.chromium.png|6200| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>

Reply:
@redongreen or @daisygeng:
Needs design review for `Table` and `Badge`.
Breaks `Badge` and `DatePicker` in Teams HC theme.

Reply:
The `DatePicker` change is actually a fix.
`Table` changes handled by @ling1726.
`Badge` changes are expected.